composer/utils/dist.py

```diff
@@ -13,6 +13,13 @@
 
 TObj = TypeVar("TObj")
 
+_NODE_RANK = "NODE_RANK"
```

Comment:
While I agree these variable names are not going to change, I would still like to follow SWE best practices here (https://softwareengineering.stackexchange.com/questions/365339/what-is-wrong-with-magic-strings), especially since the variable name is used in multiple places. I checked with others offline about their opinions on this, and it seems that constants are preferred over magic strings.

Reply:
Removed the indirection per offline thread